src/components/Footer.js

```diff
@@ -120,11 +117,20 @@ export default class Footer extends React.Component {
           onClick={isBrowser ? this.handlePlay.bind(this) : null}
           onTouchStart={!isBrowser ? this.handlePlay.bind(this) : null}
         >
-          <img
-            alt="Play Pause Button"
-            id="playButton"
-            src={this.props.playing ? PAUSE_IMAGE : PLAY_IMAGE}
-          />
+          {
+            this.props.playing
+              ?
+              <svg viewBox="0 0 640 640" width="40" height="40" fill="#fff">
```

Comment:
IMHO, it would be cleaner if we assign these to the `PAUSE_ICON` and `PLAY_ICON` constants, then use them in the ternary like so:

```
this.props.playing ? {PAUSE_ICON} : {PLAY_ICON}
```

---
Or maybe we could just create an `assets` folder and put these in there as SVG files, then import and use them here.

Reply:
Trying to do so renders the SVG as plain text. I will try to figure out how to make it work.

Reply:
@xyozio Which approach are you taking? I personally prefer the latter since, I think it's aligned with issue #29, and, uhm, I've been doing that in all of my projects 😅 But I'm not the one who decides though.

Anyway, if there is anything I can help you with, do let me know (I'm on gitter as well, btw).

Reply:
@huyenltnguyen Please check out the changes.